.circleci/config.yml

```diff
@@ -8,7 +8,7 @@ anchors:
 parameters:
   build_image_tag_ubuntu:
     type: string
-    default: slack-desktop-docker.jfrog.io/desktop-base-ubuntu-bionic:7041ce1e-376
+    default: slack-desktop-docker.jfrog.io/desktop-base-ubuntu-jammy:b2c26f2d-2038
```

Comment:
As a small heads up, this will bump Sleuth from building on Ubuntu 20 to Ubuntu 22, which means the app may not launch for some Ubuntu 20 users. Not a big deal Ubuntu 22 has been the main support for a while! But just calling in out in case there's any folks in CE who are still on 20 (or using Linux at all 😄 )